test/board_test.rb

```diff
@@ -122,4 +138,30 @@ def test_render_with_ships
     assert_equal "  1 2 3 4\nA X X X .\nB . . . .\nC . M . .\nD . . H .", board.render
     assert_equal "  1 2 3 4\nA X X X .\nB . . . .\nC . M S .\nD . . H .", board.render(true)
   end
+
+  def test_rendering_bare_board_of_large_size
+    board = Board.new(6,6)
+
+    assert_equal "  1 2 3 4 5 6\nA . . . . . .\nB . . . . . .\nC . . . . . .\nD . . . . . .\nE . . . . . .\nF . . . . . .", board.render
+  end
+
+  def test_rendering_mid_game_board_of_unusual_size
+    board = Board.new(4,5)
```

Comment:
Five columns and four rows. Like seeing this level of customization where rows and columns do not need to be equal.